test/experiment_tests.js

```diff
@@ -25,4 +24,15 @@ describe('Selector component', () => {
 			</Selector>)
 		assert.equal(selector.contains(<span>Blue</span>), true)
 	})
+
+	it ('Should render the original variant if none match', () => {
```

Comment:
this is not entirely correct. You'll want to throw the dice again if the variant is not in the list of experiments

Reply:
So if the variant is not in the list of experiments, should I reset the variant until we find one?